.github/workflows/build-wheel-macos-arm64.yaml

```diff
@@ -410,7 +410,7 @@ jobs:
     - name: Install OQC client
       if: ${{ matrix.python_version == '3.9' || matrix.python_version == '3.10'}}
```

Comment:
Can we run this with other python versions now?

Reply:
I am trying that in another PR https://github.com/PennyLaneAI/catalyst/pull/989